stellarphot/utils/data/USNO_SDSS_to_SDSS_DR7.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "USNO u'g'r'i'z' to SDSS DR7 ugriz",
+    "reference": "https://classic.sdss.org/dr7/algorithms/jeg_photometric_eq_dr1.php#usno2SDSS",
+    "from_system": {
+        "name": "USNO_SDSS",
+        "passbands": [
+            "u'",
+            "g'",
+            "r'",
+            "i'",
+            "z'"
+        ]
+    },
+    "to_system": {
+        "name": "SDSSDR7",
+        "passbands": [
+            "u",
+            "g",
+            "r",
+            "i",
+            "z"
+        ]
+    },
+
+
+    "transform_information": {
+        "name": "USNO_SDSS_to_SDSS_DR7",
+        "from_passbands": ["u'","g'","r'","i'","z'"],
+        "to_passbands": ["u","g","r","i","z"],
+        "transformation_matrix":
```

Comment:
I am not seeing where some of the numbers listed in the transformation matrix are coming from. I probably misunderstand the transformation matrix but I assume it is just these equations from the referenced 2005 website.
![image](https://github.com/user-attachments/assets/c056d8d6-1e02-473f-b82f-f890db83a072)

Reply:
I'll admit I am lost here as well as to how you get this transformation matrix from the referenced documentation.  (time passes) I looked over your gist and have a better idea of what you are doing.  You might want to distill that gist down into some comments or documentation to include with the code.